lesson_4/customerForm.js

```diff
@@ -0,0 +1,52 @@
+document.addEventListener('DOMContentLoaded', () => {
+
+  alert('!!!Увага!!! Усі дані треба вводити з заглавної літери!!!')
+
+  const userBirthDate = prompt('Якого Ви року народження?: ');
+  const userHometown = prompt('В якому місті Ви живете?: ');
+  const userFavoriteSport = prompt('Який Ваш улюблений вид спорту?: ');
+
+  const kyiv = 'Київ';
+  const washington = 'Вашингтон';
+  const london = 'Лондон';
+
+  const football = 'Футбол';
+  const box = 'Бокс';
+  const formula = 'Формула 1';
+
+  const pele = 'Пеле';
+  const tyson = 'Тайсоном';
+  const shumacher = 'Шумахером';
+
+  function sportMessage(userFavoriteSport) {
+    if (userFavoriteSport === football) {
+      return `Круто! Хочеш стати ${pele}?`;
+    } else if (userFavoriteSport === box) {
+      return `Круто! Хочеш стати ${tyson}?`;
+    } else if (userFavoriteSport === formula) {
+      return `Круто! Хочеш стати ${shumacher}?`;
+    } else {
+      return 'Щодо цього виду спорту немає інформації'
+    }
+  }
+
+  if (userBirthDate === null) {
```

Comment:
You could also use shorter version
```suggestion
  if (!userBirthDate) {
```

OR more obvious, if you feel a little bit discomfortable for now with prev variant

```suggestion
  if (!Boolean(userBirthDate)) {
```